org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/LiteTote.java

```diff
@@ -126,7 +128,9 @@ else if (thisL instanceof BaseLayer)
         while (ele.hasMoreElements())
         {
           final Editable nextE = ele.nextElement();
-          if (nextE instanceof WatchableList)
+          if (nextE instanceof LightweightTrackWrapper ||
+              nextE instanceof TrackWrapper ||
+              nextE instanceof DynamicTrackShapeSetWrapper)
           {
```

Comment:
`WatchableList` was _intended_ to be the interface that covers all the tracks.

Reply:
I am now outside home. But I think `WatchableList` included also items not time related, like Circle, Ellipse, etc. I can recheck it when I arrive home

Reply:
`DynamicTrackShapeSetWrapper` already implements `WatchableList`

![image](https://user-images.githubusercontent.com/1108513/62824592-4d6e2d00-bb98-11e9-948e-26d40e10b940.png)

Reply:
I am back home. Yes, you are correct. `LabelWrapper` and `ShapeWrapper` are also implementing `WatchableList`. If I add it, it will get highlighted, like this:

![image](https://user-images.githubusercontent.com/11049943/62825310-5cfa6f80-bbb2-11e9-95ad-d71e4711db97.png)

Reply:
@saulhidalgoaular - how about adding a test that the WatchableList has some data values (oh, and that it's visible :-) )

![image](https://user-images.githubusercontent.com/1108513/62880207-cf8b5c80-bd24-11e9-8b06-6271afe44f14.png)

Reply:
oh Good idea. It has worked!!!